tests/ls-remote.t

```diff
@@ -44,6 +44,17 @@
 
   $ git -c cinnabar.refs=bookmarks ls-remote hg::$REPO
 
+  $ git -c cinnabar.refs=bookmarks,tips ls-remote hg::$REPO
+  0000000000000000000000000000000000000000	HEAD
+  0000000000000000000000000000000000000000	refs/heads/branches/default
+  0000000000000000000000000000000000000000	refs/heads/branches/foo
+
+  $ git -c cinnabar.refs=bookmarks,heads ls-remote hg::$REPO
```

Comment:
I'd add a test with `bookmarks,heads,tips` too.

Reply:
Added this set of tests